src/Appwrite/Utopia/Response/Model/Project.php

```diff
@@ -101,6 +101,12 @@ public function __construct()
                 'default' => '',
                 'example' => '131102020',
             ])
+            ->addRule('sessionDuration', [
```

Comment:
@PineappleIOnic try to stick with existing pattern as much as possible. `authDuration`